### PR TITLE
feat: Add text input [PT-187321121]

### PR DESCRIPTION
**Pull request by dougmartin**
This adds a textarea input where any entered text is converted to a graph.

### CODE REVIEW

File: src/components/graph.tsx [hunk 1]
```diff
@@ -344,11 +344,15 @@ export const Graph = (props: Props) => {
...
+
       const d3Node: D3Node = {
         index,
         id: node.id,
-        x: node.x || 0,
-        y: node.y || 0,
+        x: node.x,
+        y: node.y,
         label: node.label,
         // radius: 15 + (5 * (node.label.length - 1)) + (5 * node.value),
```
Comment (eireland) [inline src/components/graph.tsx:357]: Do you still need this?

File: src/components/toolbar.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import "./toolbar.scss";
 export const allTools = ["select","addNode","addEdge","addText","delete","fitView","recenter","reset","home"] as const;
 const toggleableTools: Tool[] = ["select","addNode","addEdge","addText","delete"];
 const nonTopTools: Tool[] = ["reset","home"];
-const notImplementedTools: Tool[] = ["addText"];
+const notImplementedTools: Tool[] = [];
```
Comment (eireland) [inline src/components/toolbar.tsx:19]: Do you still want this?

**Review by eireland — APPROVED** [review]
There's a couple of vars that look like you don't need anymore.
Otherwise, looks good.

File: src/components/toolbar.tsx (hunk 2 above)
Comment (dougmartin) [reply src/components/toolbar.tsx:19]: Yep, I left it for any future toolbar changes.

**Pull request merged**